### PR TITLE
Adding a configurable to display x-ops-api-info

### PR DESCRIPTION
**Pull request by vinay-satish**
Signed-off-by: Vinay Satish <vinay.satish@progress.com>

### Description

Adding a new configurable variable to control the inclusion of x-ops-api-info in the response header. By default this will be false.

### Issues Resolved
Solves #2423 

### Check List

- [ ] New functionality includes tests
- [ ] All buildkite tests pass
- [ ] Full omnibus build and tests in buildkite pass
- [ ] All commits have been signed-off for the Developer Certificate of Origin. See <https://github.com/chef/chef/blob/main/CONTRIBUTING.md#developer-certification-of-origin-dco>
- [ ] PR title is a worthy inclusion in the CHANGELOG

### CODE REVIEW

File: src/oc_erchef/apps/oc_chef_wm/src/oc_chef_wm_base.erl [hunk 1]
```diff
@@ -951,9 +951,16 @@ add_api_info_header(Req, #base_state{server_api_version = RequestedVersion} = St
                               _ ->
                                   RequestedVersion
                           end,
-    Req0 = wrq:set_resp_header("X-Ops-API-Info", api_info_header_value(State), Req),
+    Req0 = add_x_ops_api_info(Req, State),
     HeaderBody = build_api_info_header_body(RequestedAPIVersion, ActualAPIVersion),
     wrq:set_resp_header("X-Ops-Server-API-Version", HeaderBody, Req0).
+-spec add_x_ops_api_info(#wm_reqdata{}, #base_state{}) ->
```
Comment (PrajaktaPurohit) [inline src/oc_erchef/apps/oc_chef_wm/src/oc_chef_wm_base.erl:957]: could we add some tests for this?

**Comment by lbakerchef** [comment]
Hello!  Can we paste umbrella tests, and examples of what happens when we use setting `true` and `false` ?

**Comment by vinay-satish** [comment]
We need to skip the test for the same in chef-zero as it does not have the `chef-server-running.json` file in it. 
The PR for the same - https://github.com/chef/chef-zero/pull/317
